spec/features/course/assessment_condition_management_spec.rb

```diff
@@ -63,8 +69,10 @@
         expect do
           find_link(nil, href: condition_delete_path).click
         end.to change { assessment.conditions.count }.by(-1)
+        title = I18n.t('activerecord.attributes.course/condition/assessment/title.complete',
+                       assessment_title: other_assessment.title)
```

Comment:
Wow I didn't know that >_< 

I'm okay with both, is the current convention to leave it out? If so, I'll remove it!